EncodingStreamingHighAvailability/media-services-high-availability-shared/Helpers/LogHelper.cs

```diff
@@ -0,0 +1,14 @@
+﻿#pragma warning disable CA1707 // Identifiers should not contain underscores
+namespace media_services_high_availability_shared.Helpers
+#pragma warning restore CA1707 // Identifiers should not contain underscores
+{
+    using Newtonsoft.Json;
+
+    public static class LogHelper
+    {
+        public static string FormatObjectForLog<T>(T objectToFormat)
+        {
+            return JsonConvert.SerializeObject(objectToFormat);
```

Comment:
Should we use Newtonsoft json converter or should we take dependecy on System.Text.Json. One reason I was using Newtosoft is to support inheritance for jobrequestmodel.

Reply:
thanks for confirmation